### ISSUE TITLE
Android Notification info screen states 'Phonegap Push' explicitly

### ISSUE THREAD

**Issue by guykatz** [phonegap/phonegap-plugin-push#2691]
### Expected Behaviour
the 'Phonegap Push' label should not be present anywhere related to the app
### Actual Behaviour
'Phonegap Push' label is present in the android notification extra info screen (appears when you usually swipe to ignore notifications many times)

#### Steps to Reproduce 
latest android 9, 
CLI 8.1.1 
plugin 2.2.3 
using FCM to send notifications
essential phone 

#### Logs taken while reproducing problem
problem describe thoroughly here (including pictures):
[https://forums.adobe.com/thread/2578937](https://forums.adobe.com/thread/2578937)

**Comment by createbyte** [comment]
Can you share a step by step approach on how you were able to get the push notifications?
Thanks

**Comment by createbyte** [comment]
I have been trying for a few days with almost zero progress. The docs are outdated and googling doesn't help much either in this regard.

**Comment by guykatz** [comment]
@createbyte I am not sure what you are requesting. are you trying to fix this bug? or just be able to send notifications?

**Comment by Stefano1964** [comment]
The label it's also displayed in the settings of the app, it's the name of the default channel created by the plugin in PushPlugin.java line 160.

**Comment by AdriVanHoudt** [comment]
As far as I can tell, the plugin always creates the default channel with that description if it is not there.
This is annoying for 2 reasons:
* Even if you delete this default channel to create your own default one, the plugin will create the not wanted default again on the next startup
	* see the only escape reason is if the default channel is there, regardless of other channels: https://github.com/phonegap/phonegap-plugin-push/blob/e5370c4bdb7f6459e82d39a7b072b10b78aa6e32/src/android/com/adobe/phonegap/push/PushPlugin.java#L154
* The default name is not usable by anyone. It would make way more sense to default to something generic like `Default channel`, `Notifications` or `[APP NAME]`
	* This behaviour is not specified in the docs and apart from demo apps I don't see anyone wanting this as their default channel description (The id is fine since this is not user facing)

I suggest to 
* Not create the default channel if there are other channels
* Rename the default channel's description to something more generic

I can do a PR for this if you want.

**Comment by guykatz** [comment]
another possibility is to ad a property for the plugin where we can overwrite this default channel name

**Comment by CSobol** [comment]
Create a channel with your own name before running `init()`, and it will be named however you choose.

**Comment by appleseedexm** [comment]
@CSobol Yes, it will create the channel with my own name, but it will still create the default channel as well and it's pretty ugly to have a Phonegap Push Channel.
Deleting afterwards is a solution, but leads to some versions of Android to show "1 deleted channel" in the app info.

I dont understand how this default channel is just forcefully created.

**Comment by CSobol** [comment]
@appleseedexm That should not be the case, and is not my experience. Are you sure you're creating the channel before you `init`?

**Comment by appleseedexm** [comment]
Yes I do, not doing so would not create the channel at all in my tests. On the other hand I can not delete the default channel before init, because that would do nothing as well. Deleting after init works, but shows up as deleted channel on some versions of Android.
